README.md

```diff
@@ -100,7 +91,7 @@ op = pydgraph.Operation(schema=schema)
 client.alter(op)
 ```
 
-Starting with Dgraph version 20.03.0, indexes can be computed in the background.
+Indexes can be computed in the background.
```

Comment:
I thought it distracted from the main point, and this would only be useful information to those using Dgraph <20.03, and I don't think we have anyone on that version now.  But if it is an issue we can reverse.